AppDashboard/static/plugins/forms/tiny_mce/plugins/table/js/table.js

```diff
@@ -242,16 +242,16 @@ function insertTable() {
 	} else
 		inst.execCommand('mceInsertContent', false, html);
 
-	tinymce.each(dom.select('table[data-mce-new]'), function(node) {
+	tinymce.each(dom.select('table[data-mce-new]'), function(node) {

```

Comment:
so what actually happened with this was my computer keyboard was lagging and somehow selected the plugins folder to commit. I tried to undo it and apparently I didn't. I made no changes with the plugins folder... how can I remove it